game-headless/build.gradle

```diff
@@ -27,6 +30,11 @@ task headlessGameArchive(type: Zip, group: 'release', dependsOn: shadowJar) {
     from(project(':game-core').file('assets')) {
         into 'assets'
     }
+    from file('scripts/run_bot')
+    from(file('scripts/run_bot.bat')) {
+        filter(ReplaceTokens, tokens: [version: version])
+        filter(FixCrLfFilter, eol: FixCrLfFilter.CrLf.newInstance("crlf")) // workaround for https://github.com/gradle/gradle/issues/1151
```

Comment:
As noted above, I couldn't use a wildcard to match the jar version in the Windows batch file as was used in the Bash script.  Instead, I'm replacing the token `@version@` during the build with the actual version.

I could have worked around this by falling back to specifying a wildcard for the classpath (`-cp "bin/*"`) and specifying the entry point explicitly (`org.triplea.game.headless.runner.HeadlessGameRunner`) instead of using the executable jar.  That seemed like a step backwards, so I went with the approach previously described.

Due to https://github.com/gradle/gradle/issues/1151, it was necessary to introduce a second filter to ensure CRLF line separators are used for the Windows batch file.  Using any filter on a Copy task (which the Zip task is using internally) has the unfortunate side-effect of replacing the line separators in the file with those of the running system.  Since our builds run on a Linux system, that means the batch file distributed with the archive would have LF line separators instead of the CRLF line separators in the Git repo.  Again, if I could use a wildcard instead of replacing the `@version@` token, none of this would be necessary.